core/src/test/scala/com/criteo/cuttle/ExecutorSpec.scala

```diff
@@ -0,0 +1,143 @@
+        failingExecutions = Seq(
+          buildExecutionForJob(fooBarJob),
+          buildExecutionForJob(fooBarJob),
+          buildExecutionForJob(untaggedJob)
+        )
+      )
+    )
+
+    println(metrics)
+
+    val expectedMetrics =
+      """# HELP cuttle_scheduler_stat_count The number of jobs that we have in concrete states
+         |# TYPE cuttle_scheduler_stat_count gauge
+         |cuttle_scheduler_stat_count {type="running"} 5
+         |cuttle_scheduler_stat_count {type="waiting"} 1
+         |cuttle_scheduler_stat_count {type="paused"} 3
+         |cuttle_scheduler_stat_count {type="failing"} 2
+         |# HELP cuttle_scheduler_stat_count_by_tag The number of executions that we have in concrete states by tag
+         |# TYPE cuttle_scheduler_stat_count_by_tag gauge
+         |cuttle_scheduler_stat_count_by_tag {tag="bar", type="paused"} 1
+         |cuttle_scheduler_stat_count_by_tag {tag="foo", type="waiting"} 1
+         |cuttle_scheduler_stat_count_by_tag {tag="foo", type="running"} 3
+         |cuttle_scheduler_stat_count_by_tag {tag="foo", type="paused"} 2
+         |cuttle_scheduler_stat_count_by_tag {tag="bar", type="failing"} 2
+         |cuttle_scheduler_stat_count_by_tag {tag="foo", type="failing"} 2
+         |cuttle_scheduler_stat_count_by_tag {tag="bar", type="running"} 1
+         |# HELP cuttle_scheduler_stat_count_by_job The number of executions that we have in concrete states by job
+         |# TYPE cuttle_scheduler_stat_count_by_job gauge
+         |cuttle_scheduler_stat_count_by_job {job="untagged_job", type="waiting"} 1
+         |cuttle_scheduler_stat_count_by_job {job="foo_bar_job", type="running"} 1
+         |cuttle_scheduler_stat_count_by_job {job="untagged_job", type="running"} 1
+         |cuttle_scheduler_stat_count_by_job {job="foo_job", type="waiting"} 1
+         |cuttle_scheduler_stat_count_by_job {job="foo_job", type="paused"} 1
+         |cuttle_scheduler_stat_count_by_job {job="untagged_job", type="failing"} 1
+         |cuttle_scheduler_stat_count_by_job {job="foo_job", type="running"} 2
+         |cuttle_scheduler_stat_count_by_job {job="foo_bar_job", type="paused"} 1
+         |cuttle_scheduler_stat_count_by_job {job="foo_bar_job", type="failing"} 2
+         |cuttle_scheduler_stat_count_by_job {job="untagged_job", type="paused"} 1
+         |# HELP cuttle_executions_total The number of finished executions that we have in concrete states by job and by tag
+         |# TYPE cuttle_executions_total counter
+         |cuttle_executions_total {type="failure", tags="bar,foo"} 1
+         |cuttle_executions_total {type="success", tags="foo"} 2
+         |cuttle_executions_total {type="success", tag="bar"} 1
+         |cuttle_executions_total {type="failure", tag="foo"} 1
+         |cuttle_executions_total {type="failure", job_id="foo_bar_job"} 1
+         |cuttle_executions_total {type="success", tag="foo"} 3
+         |cuttle_executions_total {type="success", tags="bar,foo"} 1
```

Comment:
Sorry if I was unclear but the idea is that where you have a job metric, ideally you add a `tags` label. Expected output is something like:
```
         |cuttle_executions_total {type="failure", job_id="foo_bar_job", tags="bar,foo"} 1
         |cuttle_executions_total {type="success", tag="foo"} 3```

Reply:
Ohh woops ok

Reply:
Done